vulnerability/worker.go

```diff
@@ -77,8 +79,8 @@ func NewVulnerabilityWorker(log *logp.Logger, c *config.Config) (*VulnerabilityW
 }
 
 func (f *VulnerabilityWorker) Run(ctx context.Context) {
-	// TODO: Handle deletion of snapshots
 	f.log.Info("Starting VulnerabilityWorker.work")
+	snapshots := make([]ec2.EBSSnapshot, 0)
```

Comment:
To avoid use of pointers completely, this can be made into a channel. If the main thread pops from the channel before `wg.Wait()` the channel doesn't even have to be buffered I think. I feel it would be more idiomatic that way.

Reply:
Agree, seems to be more elegant.
However, the ch has to be buffered in order for the pushing action to not be blocked.